src/main/java/core/basesyntax/dao/FruitsDaoImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.database.FruitsStorage;
+
+public class FruitsDaoImpl implements FruitsDao {
+    private static final int FRUIT_NAME_POSITION = 0;
+    private static final int FRUIT_AMOUNT_POSITION = 1;
+
+    @Override
+    public boolean add(String[] fruits) {
+        String fruit = fruits[FRUIT_NAME_POSITION];
+        int amount = Integer.parseInt(fruits[FRUIT_AMOUNT_POSITION]);
+        FruitsStorage.fruitsStorage.merge(fruit, amount, Integer::sum);
+        return FruitsStorage.fruitsStorage.get(fruit) == null;
+    }
+
+    @Override
+    public boolean remove(String[] fruits) {
```

Comment:
Standard operations in DAO: create, read, update, delete
You can use `get()` and `put()` methods instead of `add()` and `remove()`